File: Java/1015.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) throws IOException {
        Scanner scanner = new Scanner(System.in);

        double x1 = scanner.nextDouble();
        double y1 = scanner.nextDouble();
        double x2 = scanner.nextDouble();
        double y2 = scanner.nextDouble();

        System.out.printf("%.4f%n", Math.sqrt(Math.pow(x2 - x1, 2) + Math.pow(y2 - y1, 2)));
    }
}

File: 1. Beginner/1015/1015.c
#include <stdio.h>
#include <math.h>

int main() {
    float x1, y1, x2, y2, r;
    scanf("%f %f %f %f", &x1, &y1, &x2, &y2);
    r = sqrt(pow(x2 - x1, 2) + pow(y2 - y1, 2));
    
    printf("%.4f\n", r);

    return 0;
}